MUXSDKStats/MUXSDKStats/MUXSDKPlayerBindingManager.h

```diff
@@ -0,0 +1,29 @@
+//
+//  MUXSDKPlayerBindingManager.h
+//  MUXSDKStats
+//
+//  Created by Nidhi Kulkarni on 1/30/20.
+//  Copyright © 2020 Mux, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#if TARGET_OS_IOS
+@import MuxCore;
+#else
+@import MuxCoreTv;
+#endif
+#import "MUXSDKPlayerBinding.h"
+#import "MUXSDKCustomerPlayerDataStore.h"
+#import "MUXSDKCustomerVideoDataStore.h"
+
+@interface MUXSDKPlayerBindingManager : NSObject<MUXSDKPlayDispatchDelegate>
+
+@property (nonatomic, weak) id<MUXSDKCustomerPlayerDataStoring> _Nullable customerPlayerDataStore;
+@property (nonatomic, weak) id<MUXSDKCustomerVideoDataStoring> _Nullable customerVideoDataStore;
+@property (nonatomic, weak) NSDictionary * _Nullable viewControllers;
+@property (nonatomic, retain) Class _Nullable core;
+
+- (void) initializeCoreForPlayer:(NSString *_Nonnull) name;
```

Comment:
Open to other/clearer names for this method. The idea is that this method is responsible for making sure MuxCore gets the initial sequence of events in the correct order

Reply:
One option is `newViewForPlayer:` or `startNewViewForPlayer`